wordpress-mobile/WordPress-Android
# Feature/login helpshift tags

This PR updates the Helpshift `origin` tags passed by the new login flow, and adds a new tag to flag when the login flow is being used to connect a site to Jetpack. 

To test:
Scenario 1:
- Confirm that tags are properly passed via the Help button, and via the "more" button on the site address help dialog. 

Scenario 2: 
- Add a self-hosted site to the app. 
- On the stats screen, connect to Jetpack. 
- Before completing the login flow, tap the Help button and confirm that the new Jetpack key is being properly passed. 
- Start a new Helpshift conversation.
- Sign in to Helpshift and confirm that the origin and Jetpack keys are properly passed.  You should see something like this : 
<img width="593" alt="screen shot 2017-07-30 at 5 39 28 pm" src="https://user-images.githubusercontent.com/1435271/28757754-7ee5be0c-754f-11e7-972c-339acae10359.png">

Needs review: @hypest would you do the honors sir?
